src/test/java/nurseybook/logic/commands/EditTaskCommandTest.java

```diff
@@ -239,6 +242,18 @@ public void execute_editDateOfTask_reordersTaskList() {
         assertCommandSuccess(editTaskCommand, model, expectedCommandResult, expectedModel);
     }
 
+    @Test
+    public void execute_editDateOfRecurringTaskToPastDate_throwsCommandsException() {
+        showTaskAtIndex(model, INDEX_FOURTH);
+
+        Task taskInFilteredList = model.getFilteredTaskList().get(INDEX_FIRST.getZeroBased());
+        Task editedTask = new TaskBuilder(taskInFilteredList).withDate(VALID_DATE_NOV).build(); // past current date
+
+        EditTaskCommand editTaskCommand = new EditTaskCommand(INDEX_FIRST,
+                new EditTaskDescriptorBuilder(editedTask).build());
+        assertCommandFailure(editTaskCommand, model, MESSAGE_INVALID_TASK_DATETIME_FOR_RECURRING_TASK);
+    }
+
```

Comment:
This way of checking for past date would mean that once VALID_DATE_NOV passes, the test case would fail. Perhaps you could get current date with `LocalDate.now()`, minus 1 day from it with `.minusDays(1)`, and then use that to test.

Reply:
Resolved, misinterpreted it.